spec/unit/jobs/enqueuer_spec.rb

```diff
@@ -11,11 +11,13 @@ module VCAP::CloudController::Jobs
         jobs: {
           global: {
             timeout_in_seconds: global_timeout,
-          }
+          },
+          priorities: job_display_name
```

Comment:
As the `priorities` node is optional, this line could be changed to `**priorities`...